src/XenobladeX/Mods/BattleEscapeDistance/rules.txt

```diff
@@ -3,7 +3,10 @@ titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
 name = "Escape distance reducer"
 path = "Xenoblade Chronicles X/Mods/Battle/Escape distance reducer"
 description = Reduce escape distance needed to lose aggro from enemies.
-version = 4
+version = 5
+
+[Default]
+$mod = 1.0
```

Comment:
probably should add a preset like "Don't apply" so that this 1.0 gets used

Reply:
I don't understand the need to add a "Default" or "Don't apply" preset in all the graphic packs, because if you want the default value then you simply don't activate the pack, right?
The only reason I add the [Default] setting is because without that the pack doesn't load.